src/Dialogs/Settings/dlgConfiguration.cpp

```
@@ -124,6 +131,22 @@ static constexpr TabMenuPage look_pages[] = {
   { nullptr, nullptr }
 };
 
+#if defined(HAVE_PCMET) || defined(HAVE_HTTP)
+static constexpr TabMenuPage weather_pages[] = {
+#ifdef HAVE_HTTP
+  { N_("Thermal Information Map"), CreateWeatherConfigPanel },
+#endif
+  { "RASP", CreateRaspConfigPanel },
+#ifdef HAVE_PCMET
+  { "Flugwetter (pc_met)", CreatePCMetConfigPanel },
```

Cppcheck (2.21.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)